code/ensemble/utils/run-multiple-ensembles.R

```diff
@@ -18,6 +18,7 @@ source(here("code", "ensemble", "utils", "run-ensemble.R"))
 run_multiple_ensembles <- function(forecast_dates,
                                    methods,
                                    exclude_models = NULL,
+                                   exclude_designated_other = TRUE,
```

Comment:
I think this function only needs `forecast_dates` and `methods` as arguments. Everything else could be passed via `...`

```
run_multiple_ensembles <- function(forecast_dates, 
                                   methods, 
                                   ...) {
```

and later

```
  ensembles <- map2(methods,
                    dates,
                    ~ safe_run_ensemble(method = .x,
                                        forecast_date = .y,
                                        ...)
```

Reply:
Ah, yes that would be better - thank you!